corehq/apps/hqwebapp/urls.py

```diff
@@ -3,8 +3,9 @@
 
 urlpatterns = patterns(
     'corehq.apps.hqwebapp.views',
+    (r'^$', 'redirect_to_default'),
     url(r'^homepage/$', 'redirect_to_default', name='homepage'),
-    url(r'^home/$', 'landing_page', name='landing_page'),
+    url(r'^default_landing/$', 'landing_page', name='landing_page'),
```

Comment:
if you have `default_landing` you are not running the prelogin site / haven't pulled the repo